src/borg/lrucache.py

```diff
@@ -28,6 +28,14 @@ def __delitem__(self, key):
     def __contains__(self, key):
         return key in self._cache
 
+    def get(self, key, default=None):
+        value = self._cache.get(key, default)
+        if value is default:
+            return value
```

Comment:
sure it makes sense if None could be part of the legitimate values.

if you do not want to allow that, document and assert it at least.